pjlib/src/pj/ioqueue_winnt.c

```diff
@@ -378,11 +412,24 @@ PJ_DEF(pj_status_t) pj_ioqueue_create2(pj_pool_t *pool,
     rc = sizeof(union operation_key);
 
     /* Check that sizeof(pj_ioqueue_op_key_t) makes sense. */
-    PJ_ASSERT_RETURN(sizeof(pj_ioqueue_op_key_t)-sizeof(void*) >= 
+    PJ_ASSERT_RETURN(sizeof(pj_ioqueue_op_key_t)-3*sizeof(void*) >= 
```

Comment:
Why 3 times? Perhaps some doc would help, so we can easily recall in the future.